### ISSUE TITLE
Ability to reorganize filters in questions and dashboards

### ISSUE THREAD

**Issue by rafaelveloso** [metabase/metabase#4780]
One small but nice-to-have feature in Metabase would be the ability to re-organize the way filters show on top of questions and dashboards. 
If, as an example, I have a dashboard with 3 hierarchically organized filters (`state`, `zipcode`, `city` in this order) and I want to add a new `country` filter, it would be more intuitive to the end-user if `country` was displayed on the left-hand side of all other pre-existent filters.

Bottom line, once filters are created, there's no way (at least to my knowledge) to re-organize how they're displayed. The only way I know to do it at the moment is to remove all filters and recreate them (which might be painful depending on how many cards are on the dashboard)

**Comment by tlrobinson** [comment]
Duplicate of #4776

**Comment by mazameli** [comment]
…which itself was a duplicate of https://github.com/metabase/metabase/issues/3238
